client_dashboard/common/src/commonMain/kotlin/org/thechance/common/data/remote/gateway/RemoteGateway.kt

```diff
@@ -33,19 +45,29 @@ class RemoteGateway(
         )
     }
 
-    override suspend fun getTaxis(): List<Taxi> {
-        return emptyList()
+    override suspend fun getTaxis(page: Int, numberOfUsers: Int): DataWrapper<Taxi> {
+        return DataWrapper(
+            totalPages = 0,
+            numberOfResult = 0,
+            result = emptyList(),
+        )
     }
 
-    override suspend fun createTaxi(taxi: AddTaxi): Taxi {
+
+    override suspend fun createTaxi(taxi: NewTaxiInfo): Taxi {
         println("createTaxi: $taxi")
         return Taxi("1", "1", CarColor.BLACK, "1", 4, "1", TaxiStatus.OFFLINE, "1")
     }
 
-    override suspend fun findTaxiByUsername(username: String): List<Taxi> {
-        return emptyList()
+    override suspend fun findTaxiByUsername(username: String, page: Int, offset: Int): DataWrapper<Taxi> {
+        return DataWrapper(
+            totalPages = 0,
+            numberOfResult = 0,
+            result = emptyList(),
+        )
     }
 
+
     override suspend fun getPdfTaxiReport() {
         //todo
```

Comment:
adding it like this might not notify us when a runtime error or git push happens
the correct way is
// TODO

Reply:
![image](https://github.com/TheChance101/beep-beep/assets/105972942/51a3bcec-6336-44d0-9242-249869367d5f)

todo work too